modules/vcd_vapp_vm.py

```diff
@@ -342,6 +343,7 @@ def add_vm(self):
         ip_allocation_mode = params.get('ip_allocation_mode')
         cust_script = params.get('cust_script')
         storage_profile = params.get('storage_profile')
+        compute_policy_href = params.get('compute_policy_href')
```

Comment:
Do we need to get `compute_policy_href` in `add_vm`?

Reply:
True, I have now removed it.